Temporarily remove references to Grafana.com (#279)
openservicemesh/osm-docs: content/docs/guides/observability/metrics.md

##### Importing OSM Dashboards

> OSM Dashboards are available both through the [OSM repository](https://github.com/openservicemesh/osm/tree/release-v0.9/charts/osm/grafana), and are importable as json blobs through the web admin portal.

Reviewer: ```suggestion
OSM Dashboards are available through [our repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards), which can be imported as json blobs on the web admin portal.
```
